btk/obs_conditions.py

```diff
@@ -114,6 +114,34 @@ def get_psf_sky(self, psf_stamp_size):
         return psf_image, mean_sky_level
 
 
+class CosmosCutout(Cutout):
+    @staticmethod
+    def psf_function(r):
+        # usually r = 0.3 * pix
+        return galsim.Moffat(2, r)
+
+    def get_psf(self, psf_stamp_size):
+        assert psf_stamp_size % 2 == 1
+        psf_int = self.psf_function(self.psf_args).withFlux(1.0)
+
+        # Draw PSF
+        psf = psf_int.drawImage(
```

Comment:
Be careful about pixel integration. Make sure that it is done only once, either here or during the drawing of the final image.